region/norfair/east.json

```diff
@@ -5151,10 +5160,22 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 820},
-                    {"lavaFrames": 150},
-                    {"spikeHits": 7}
-                  ]
+                    {"heatFrames": 520},
+                    {"lavaFrames": 80},
+                    {"spikeHits": 3},
+                    {"or": [
+                      {"and": [
+                        "canIframeSpikeJump",
+                        "canDamageBoost"
+                      ]},
+                      {"and": [
+                        {"spikeHits": 4},
+                        {"heatFrames": 300},
+                        {"lavaFrames": 70}
+                      ]}
+                    ]}
+                  ],
+                  "note": "Delay the damageboost from the spikes slightly in order to rise above the lava before moving."
```

Comment:
```suggestion
                  "note": "Damageboosts can be used to save energy - delay the damageboost from the spikes slightly in order to rise above the lava before moving."
```

Reply:
Something like this could be helpful so that the player isn't confused if they have dboosts turned off